provider/pkg/provider/ids.go

```diff
@@ -115,6 +115,24 @@ func buildURL(
 	return uri.String(), nil
 }
 
+func getInputString(inputs resource.PropertyMap, key resource.PropertyKey) (string, bool) {
```

Comment:
It looks like this function is doing 3 few things:
1. Get input value
2. Check if the value exists
3. Check if it can be parsed if it exists

In Kubernetes packages, I'm used to this having 3 return types:
`(string, bool, error)`
So return false if value is not found and an error if we can't convert it to string. Would it be worthwhile to expand this to 3 return types?

Reply:
Nice, done 👍